code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -614,24 +614,18 @@
 	reagents.chem_temp = COOKTEMP_READY
 
 /obj/item/weapon/reagent_containers/food/drinks/dry_ramen
-	name = "\improper cup ramen"
+	name = "\proper cup ramen"
```

Comment:
Cup ramen is improper. (the big brand name for cup instant ramen is Cup Noodles)

Proper: "User has thrown cup ramen."
Improper: "User has thrown the cup ramen."

Reply:
if you say so